app/scripts/views/components/vc/addVCPopup.js

```diff
@@ -47,6 +47,10 @@
             self_.addNewVc = false;
             jQuery.notify('Added to new Virtual Study', 'success');
           });
+        } else {
+          // TODO: if we need to consider whether vcSession is available,
+          // should we have similar notify as well like
+          // jQuery.notify('Session service is not available', 'Warning');
```

Comment:
Since this component is part of vcSession we don't need to check whether the object is preset or not. However this component wont be used in near future (adding a particular sample from the query results(oncoprint, mutations etc.) to a cohort).